.github/workflows/codecov.yml

```diff
@@ -0,0 +1,13 @@
+name: Codecov
+on: [ push, pull_request ]
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - uses: codecov/codecov-action@v4
+        with:
+          token: ${{ secrets.CODECOV_TOKEN }}
+          files: ./kover/coverage.xml
```

Comment:
Is the coverage file being treated as generated code in PR review process already? If not, that should be added to the .gitattributes file so it doesn't create additional noise.

Reply:
Good callout, thanks. We were previously checking it in. Removed coverage reports from the repo, updated our .gitignore, and modified our workflows accordingly.